components/org.wso2.carbon.identity.auth.abstractotp/src/main/java/org/wso2/carbon/identity/auth/abstractotp/PasswordlessOTPAuthenticator.java

```diff
@@ -0,0 +1,7 @@
+package org.wso2.carbon.identity.auth.abstractotp;
+
+/**
+ * This is the interface to be implemented if an OTP authenticator need to support for passwordless authentication.
+ */
+public interface PasswordlessOTPAuthenticator {
```

Comment:
What is the value of having this interface?

Reply:
By having this interface we can let the authenticators that  extend the AbstractOTPAuthenticator be MFA authenticators without supporting 1FA.